assets/content/cookbook/Other/search-for-files-with-extension.md

```diff
@@ -0,0 +1,38 @@
+# Search for all files of particular extension
+
+This example uses Haxe's Filesystem to recursively search for all files of certain extension. 
+
+```hx
+static function recursiveSearch(directory:String = "path/to/", extension:String) {
+    var files:Array<String> = [];
+    
+    if (sys.FileSystem.exists(directory)) {
+      for (file in sys.FileSystem.readDirectory(directory)) {
+        var path = haxe.io.Path.join([directory, file]);
+        if (!sys.FileSystem.isDirectory(path)) {
+        // regex expression to search for files with a '.'
```

Comment:
The indentation seems to be off here? (the if is on the same level as its body)

Reply:
@Gama11  I have made changes, please check again.

Thanks.

Reply:
Something still seems off, I'm seeing 4 space indentation in some cases and 2 in others. Looks like the readme wants you to use 2 spaces, maybe run haxe-formatter over it with that? https://github.com/HaxeFoundation/code-cookbook#formatting

Reply:
@Gama11 awkward, I will try again